src/modules/task/internal/store/task-repository.ts

```diff
@@ -0,0 +1,24 @@
+import mongoose, { CallbackError, Connection } from 'mongoose';
+import ConfigService from '../../../config/config-service';
+import { TaskDb, taskDBSchema } from './task-db';
+
+export default class TaskRepository {
+  public static task: mongoose.Model<TaskDb>;
```

Comment:
this should be `taskDB`. Can you please ensure it is consistent with account module

Reply:
@jjalan  sir, this was previously ```TaskDB``` but in previous review I was told to change it to ```taskDb``` but linting does not allow this naming convention and linting shows to use PascalCase, so I used this ```TaskDb```.

Reply:
Can you please work with @akashlakhera  to find a consistent convention?